security/client/ipsec.go

```diff
@@ -124,6 +125,23 @@ func do_ipsec(conn grpc.ClientConnInterface, ctx context.Context) {
 	}
 	log.Printf("Returned connections: %v", list_certs_ret)
 
+	// Ping across the tunnel.
+	// .NOTE: The container this test runs in is linked to the appropriate
+	//        strongSwan container, and we hardcode the IPv4 address here.
+	pinger, err := ping.NewPinger("10.3.0.1")
```

Comment:
A better approach would be to use an environment variable that can be set in the compose file.

Reply:
in storage service I use this snippet, for example:
```
var (
	port = flag.Int("port", 50051, "The server port")
)
```

Reply:
Good ideas, I fixed this.